stripe-samples/connect-onboarding-for-standard
# Add Go version

This PR implements the Go version of the server.

I followed what was done in the [Node.js version](https://github.com/stripe-samples/connect-onboarding-for-standard/blob/master/server/node/server.js) and converted the code to Go.

I tried to write some tests but it was unsuccessful. I was hoping I could find another sample in which we added tests for Go so I could start from that but it looks like we don't have any(?) If it's a problem, let me know and I can spend more time trying to figure it out 🙂 .

### Test plan:
* Fetch this branch
* Run `go run server.go`
* [ ] A server should start on port 4242
* [ ] Clicking on the button in the UI should redirect you to a `https://connect.stripe.com/` URL
* [ ] On refresh, you should see the same page
